sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -194,8 +195,19 @@ private void onInitialized(
             mHandler.post(new Runnable() {
                 @Override
                 public void run() {
+                    // call onInitialized on each callbacks
                     for (DeployGateCallback callback : mCallbacks) {
                         callback.onInitialized(true);
+                    }
+                    for (DeployGateInitializeCallback callback : mInitializeCallbacks) {
+                        callback.onInitialized(true);
+                    }
+
+                    // after call onInitialized, then call onStatusChanged
+                    for (DeployGateCallback callback : mCallbacks) {
+                        callback.onStatusChanged(isManaged, isAuthorized, loginUsername, isStopped);
+                    }
+                    for (DeployGateStatusChangeCallback callback : mStatusChangeCallbacks) {
                         callback.onStatusChanged(isManaged, isAuthorized, loginUsername, isStopped);
                     }
```

Comment:
Keep order of call callback methods.
We call `onInitializad()` first, after that call `onStatusCahnged()`.